cypress/integration/bulk-edit/csv/bulk-edit-csv-user-barcodes.js

```diff
@@ -36,6 +38,10 @@ describe('bulk-edit', () => {
       FileManager.deleteFile(`cypress/fixtures/${userBarcodesFileName}`);
```

Comment:
Do you remove this file (cypress/fixtures/${invalidUserBarcodesFileName})  after the test execution ?

Reply:
done